flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterClientV2.java

```diff
@@ -146,7 +146,9 @@ public ApplicationStatus getApplicationStatus() {
 
 	@Override
 	public void finalizeCluster() {
-		// Do nothing
```

Comment:
I have reverted the changes and fixed another problem as Ted Yu metioned in FLINK-5488, thanks @zentol